packages/ddex/processor/src/db.ts

```diff
@@ -343,6 +355,24 @@ export const releaseRepo = {
   },
 }
 
+//
+// kv repo
+//
+
+export const kvRepo = {
```

Comment:
atm just holds a cookie secret value which should be unique per server... but saves the operator from having to set `COOKIE_SECRET='openssl rand -hex 16'` in the `.env` manually.  One less env variable to worry about.  It gets initialized if it doesn't already exist.

I don't have any other use cases atm, but since all db access happens thru dedicated methods in `db.ts` every specific data type can have dedicated access functions so that there's not just arbitrary usage happening everywhere.

ddex can still reasonably run with no docker dependencies... so in some ways the table can serve as a redis substitute.